extensions/positron-r/package.nls.json

```diff
@@ -59,5 +59,10 @@
 	"r.configuration.pipe.native.description": "Native pipe available in R >= 4.1",
 	"r.configuration.pipe.magrittr.description": "Pipe operator from the magrittr package, re-exported by many other packages",
 	"r.configuration.diagnostics.enable.description": "Enable R diagnostics globally",
-	"r.configuration.taskHyperlinks.description": "Turn on experimental support for hyperlinks in package development tasks"
+	"r.configuration.taskHyperlinks.description": "Turn on experimental support for hyperlinks in package development tasks",
+	"r.configuration.defaultRepositories.description": "The default repositories to use for R package installation, if no repository is otherwise specified in R startup scripts (restart Positron to apply).\n\nThe default repositories will be set as the `repos` option in R.",
+	"r.configuration.defaultRepositories.auto.description": "Automatically choose a default repository, or use a repos.conf file if it exists.",
+	"r.configuration.defaultRepositories.rstudio.description": "Use the RStudio CRAN mirror (cran.rstudio.com)",
```

Comment:
Taking this chance to have a side discussion:

In my `.Rprofile`, I have

```
...
repos = c(
    P3M = "https://p3m.dev/cran/latest",
    CRAN = "https://cloud.r-project.org"
)
...
```

In terms of what we feature, is there a reason to push cran.rstudio.com over cloud.r-project.org? I guess I've implicitly preferred the latter from some super vague reasons around rstudio->posit rebrand and avoiding overt branding.

Reply:
I don't think we necessarily want to push cran.rstudio.com (Posit also sponsors `cloud.r-project.org`), I just used it to match ark's existing behavior. Might be worth providing a `cloud.r-project.org` option too, let's see what folks say.